IFC4x3/Properties/m/MandatoryLimitDbValue_3Dyb6v_Lb0uePKj$y26Fb4/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="MandatoryLimitDbValue_3Dyb6v_Lb0uePKjy26Fb4" Name="MandatoryLimitDbValue" UniqueId="cdf251b9-f959-40e2-8654-b7ff0218f944" PrimaryDataType="IfcSoundPowerLevelMeasure" />
```

Comment:
Firstly cannot have a unit in name, mandatory implies an external design criteria where as i would expect this property on the PSet it is on to be the performance value of the equipment. this should be harmonised with `NoiseLimitValue` also